readme.md

```diff
@@ -218,13 +223,20 @@ Some things to consider:
 2. What were the necessary commands to run to incorporate those changes?
 3. What kind of system and channels best allow developers to prevent, and resolve merge conflicts most effectively?
 
-##[Pennies!](https://github.com/ga-wdi-exercises/pennies) (20 minutes)
+## [Pennies!](https://github.com/ga-wdi-exercises/pennies) (20 minutes)
 
```

Comment:
take pennies out. there's no way they can do this on their own and i'm 70 percent sure they did this in erik's agile lesson